src/components/SettlementButton.js

```diff
@@ -164,42 +159,19 @@ function SettlementButton({
 
         // To achieve the one tap pay experience we need to choose the correct payment type as default,
         // if user already paid for some request or expense, let's use the last payment method or use default.
-        let paymentMethod = nvp_lastPaymentMethod[policyID] || '';
-        if (!shouldShowPaymentOptions) {
-            if (!paymentMethod) {
-                // In case the user hasn't paid a request yet, let's default to VBBA payment type in case of expense reports
-                if (isExpenseReport) {
-                    paymentMethod = CONST.IOU.PAYMENT_TYPE.VBBA;
-                } else if (canUseWallet) {
-                    // If they have Wallet set up, use that payment method as default
-                    paymentMethod = CONST.IOU.PAYMENT_TYPE.EXPENSIFY;
-                } else {
-                    paymentMethod = CONST.IOU.PAYMENT_TYPE.ELSEWHERE;
-                }
-            }
-
-            // In case of the settlement button in the report preview component, we do not show payment options and the label for Wallet and ACH type is simply "Pay".
-            return [
-                {
-                    ...paymentMethods[paymentMethod],
-                    text: paymentMethod === CONST.IOU.PAYMENT_TYPE.ELSEWHERE ? translate('iou.payElsewhere') : translate('iou.pay'),
-                },
-            ];
-        }
+        const paymentMethod = nvp_lastPaymentMethod[policyID] || '';
         if (canUseWallet) {
             buttonOptions.push(paymentMethods[CONST.IOU.PAYMENT_TYPE.EXPENSIFY]);
         }
-        if (isExpenseReport) {
-            buttonOptions.push(paymentMethods[CONST.IOU.PAYMENT_TYPE.VBBA]);
-        }
+        buttonOptions.push(paymentMethods[CONST.IOU.PAYMENT_TYPE.VBBA]);
```

Comment:
This change caused [duplicated "Pay with Expensify" option](https://github.com/Expensify/App/issues/31829)